### PR TITLE
Suggested changes for update tables

### PR DESCRIPTION
**Pull request by robertpp**
- [x] Removed 'update history' from the title of the section. Example given for Skype for Business Server 2019 only. The article is about update history and the string is repetitive in the table of contents on the right. Requires an update of the hrefs to the section. If accepted, it might be applied consistently to the other sections.
- [x] Added the link to the standard document for Skype for Business Server 2019 updates. Simplified the text to match the one from the Skype for Business Server 2015 section.
- [x] Removed the reference to Office 365. This belongs to the client update history document.
- [x] Rearranged the table to include the version number and order with the latest update at the top.
- [x] Kept the reference to the main KB only in the latest update. Usually, PG does not publish a dedicated KB for each release, but merely updates the standard one each time. This makes the column useless, unless all components are listed, which do have dedicated KBs. It also means information about what component was updated in each CU is lost after each release. Tried to add it as a dedicated column, but makes the column too large compared to the other columns. The information is very useful in practice as the IT admins do not have an official source of information for the components update history.

Example:

|Package Name|Version|Release Date|Components|
|:--- |:--- |:--- |:--- |
|Cumulative Update 1 Hotfix 1| 17.0.2046.143 | September 2019 |[Core Components](https://support.microsoft.com/kb/4499964), [Unified Communications Managed API 5.0 Runtime](https://support.microsoft.com/kb/4499966), [Web Components Server](https://support.microsoft.com/kb/4499967), [Front End Server and Edge Server](https://support.microsoft.com/kb/4499991), [Backup Service, Application Host](https://support.microsoft.com/kb/4499965), [Centralized Logging Service](https://support.microsoft.com/kb/4499974), [Core Management Server](https://support.microsoft.com/kb/4499978), [Replica Replicator Agent](https://support.microsoft.com/kb/4499980), [Enterprise Web App](https://support.microsoft.com/kb/4499968), [Conferencing Server](https://support.microsoft.com/kb/4499969), [Conferencing Attendant](https://support.microsoft.com/kb/4499972), [Conferencing Announcement](https://support.microsoft.com/kb/4499973), [Call Park service](https://support.microsoft.com/kb/4499975), [Web Conferencing Server](https://support.microsoft.com/kb/4499976), [Mediation Server](https://support.microsoft.com/kb/4499977), [Bandwidth Policy Service](https://support.microsoft.com/kb/4499979), [Response Group Service](https://support.microsoft.com/kb/4499981), [Video Interop Server](https://support.microsoft.com/kb/4499983), [Audio Test service](https://support.microsoft.com/kb/4499971)|
|Cumulative Update 1| 17.0.2046.123 | July 2019 | |
| | | | |

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/sfb-server-updates.md [hunk 1]
```diff
@@ -15,19 +15,19 @@ description: "Find and manage updates for Skype for Business Server 2015/2019 an
...
+Find and manage updates for [Skype for Business Server 2019](#skype-for-business-server-2019), [Skype for Business Server 2015](#skype-for-business-server-2015-update-history),  [Lync Server 2013](#lync-server-2013-update-history),  and [Lync Server 2010](#lync-server-2010-update-history) in one place. Use the links on this page to get more information about updates, and then download the updates.
 
 For the latest Skype for Business Online and Office 365 updates, see the [Microsoft 365 Roadmap](https://go.microsoft.com/fwlink/p/?linkid=529454).
 
-## Skype for Business Server 2019 update history
+## Skype for Business Server 2019
 
-KB 4470124 is the master KB for Skype for Business Server 2019 which has all the historical list of all the Cumulative Update and Hotfixes that have been published. 
+[KB 4470124](https://support.microsoft.com/kb/4470124) contains all Cumulative Updates that have been released for Skype for Business Server 2019.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/sfb-server-updates.md]:
```suggestion
[KB 4470124](https://support.microsoft.com/kb/4470124) contains all the Cumulative Updates that have been released for Skype for Business Server 2019. 
```

**Comment by scanum** [comment]
@Ariantor PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
PR is based on an old version of the topic. I will submit the suggested changes to the writer.

**Pull request closed without merging**